src/MassTransit.Abstractions/Attributes/MessageUrnAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace MassTransit.Attributes
+{
+    using System;
+
+    /// <summary>
+    /// Specify the URN used for this message contract
+    /// if configured.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public class MessageUrnAttribute :
```

Comment:
Would need the documentation updated as well: https://masstransit.io/documentation/concepts/messages#message-attributes

(in the /doc folder of the repository)

Reply:
I've added documents in two locations I through were relevant.